src/backend/main.ts

```diff
@@ -1676,7 +1676,7 @@ ipcMain.on('processShortcut', async (e, combination: string) => {
       break
     // hotkey to open the settings on frontend
     case 'ctrl+k':
-      sendFrontendMessage('openScreen', '/settings/app/default/general')
+      sendFrontendMessage('openScreen', '/settings/general')
```

Comment:
we don't need these `app/default` routes anymore, the Settings component always renders Heroic's settings, game's settings are not handled by this url